roles/precis/tasks/main.yml

```diff
@@ -1,4 +1,19 @@
 ---
+- name: Create cep service
+  os_keystone_service:
+    name: cep
```

Comment:
You will probably need to have `auth` (or `openstack_auth`, I forget which) here. Otherwise, it will be writing updates to the secondary Keystone server and cause replication errors. There is an `openstack_auth` var already set, I think, which you can use here.